### ISSUE TITLE
[1.14] Optifine Crash

### ISSUE THREAD

**Issue by ghost** [mezz/JustEnoughItems#1668]
So JEI 1.14.4.6.0.0.10
Forge 28.0.45
Optifine preview F4 pre 2 - I know but I like shaders

Crash Report https://pastebin.com/H5HmtrJf

The world loads and can run/skip and dance, but when Press 'E' game crashes
With Optifine removed its ok

Is this a JEI Fix or shall I report to Optifine

**Comment by mellow2012** [comment]
> So JEI 1.14.4.6.0.0.10
> Forge 28.0.45
> Optifine preview F2 - I know but I like shaders
>
> Crash Report https://pastebin.com/H5HmtrJf
> 
> The world loads and can run/skip and dance, but when Press 'E' game crashes
> With Optifine removed its ok
> 
> Is this a JEI Fix or shall I report to Optifine


here too, when press "E" the game crashed

**Comment by pentyum** [comment]
Same problem, and my optifine is preview_OptiFine_1.14.4_HD_U_F4_pre2.jar

**Comment by GrumpyZade** [comment]
Got so happy when the game loaded, then very sad when I pressed E ;(

**Comment by ghost** [comment]
As a temp solution just remove Optifine

**Comment by GrumpyZade** [comment]
Imo I prefer having optifine more than jei rn just because I don't have enough mods installed that add recipes. I'll just continue to patiently wait lol

**Comment by mezz** [comment]
Thanks for the report!
Please report this to Optifine here: https://github.com/sp614x/optifine/issues
JEI is not really rendering anything differently between 1.12 and 1.14 so I'm not sure why it doesn't still work.

**Comment by mezz** [comment]
I will follow the new Optifine issue here: https://github.com/sp614x/optifine/issues/2748

**Comment by PseudoResonance** [comment]
Looks like this is likely an OptiFine issue and JEI just happens to make it noticeable, as shown here: sp614x/optifine#2737

**Comment by Orange1861** [comment]
OptiFine 1.14.4_HD_U_F2 is not compatible with forge, I'd recommend closing the issue.

**Comment by mezz** [comment]
Thanks for the info.
I am leaving this open for now so people can find it more easily, to help reduce duplicate issues reported here.

**Comment by PseudoResonance** [comment]
The original question was about F2, but it's kinda just become a general OptiFine and other mods issue, and more specifically OptiFine F4 prerelease 2.

**Comment by ghost** [comment]
> 
> 
> OptiFine 1.14.4_HD_U_F2 is not compatible with forge, I'd recommend closing the issue.

I got the version wrong, it was 1.14.4 F4 pre 2 which is compatible with forge - I rechecked the error log I posted

**Comment by F0xin1** [comment]
Same issue here. Initial logs looked like it was to do with FastRender, tried disabling it in optifine settings but the same error occurs.

```
The game crashed whilst rendering ingredient
Error: java.lang.IllegalAccessError: tried to access method net.minecraft.client.renderer.ItemRenderer.func_191961_a(Lnet/minecraft/client/renderer/model/IBakedModel;Lnet/minecraft/item/ItemStack;)V from class mezz.jei.render.ItemStackFastRenderer
```

**Comment by sp614x** [comment]
OptiFine 1.14.4_HD_U_F4_pre3
 - fixed Forge access transformers (#2737)

**Comment by Ommina** [comment]
Thank you for the update, @sp614x .  Now to see who won the pool for "how many duplicate issue reports will there be?".
